compiler/rustc_target/src/spec/mod.rs

```diff
@@ -2137,6 +2141,16 @@ impl Target {
         ))
     }
 
+    /// Load a built-in target
+    pub fn builtin(target_triple: &TargetTriple) -> Result<Target, String> {
```

Comment:
```suggestion
    pub fn expect_builtin(target_triple: &TargetTriple) -> Target {
```
The only user will panic on non-builtin anyway, so we can panic inside this function instead.